examples/test/post-processing/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Post Process</title>
+    <meta name="description" content="Post process — A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+    <script src="components/pulse.js"></script>
+    <script src="components/envmap.js"></script>
+    <script src="components/bloom.js"></script>
+    <script src="components/compositor.js"></script>   
+  </head>
+  <body>
+    <a-scene post-process="">
+      <a-assets>
+        <video id="video" src="https://ucarecdn.com/bcece0a8-86ce-460e-856b-40dac4875f15/"
+               autoplay loop></video>
+      </a-assets>
+      <a-sphere envmap="" pulse="phase:1.58" rotation="0 180 0" color="white" position="0 -50 -160" radius="40"></a-sphere>
+      <a-videosphere pulse="phase:0; exponent:5;" src="#video" rotation="0 180 0"></a-videosphere>
+    </a-scene>
+      
+    <script>
+        var scene = document.querySelector("a-scene");
```

Comment:
Not sure I got that, you mean renaming the file and component or something else?